frontends/p4/typeMap.cpp

```diff
@@ -333,7 +335,10 @@ int TypeMap::minWidthBits(const IR::Type* type, const IR::Node* errorPosition) {
         return 0;
     }
 
-    ::error("%1%: width not well-defined", errorPosition);
+    if (errorPosition)
```

Comment:
errorPosition should never be nullptr, so you can replace this with CHECK_NULL(errorPosition);

Reply:
My first use of midWidthBits was in my getTypeWidth() function where I thought the errorPosition was not avaiable.  So I sent a nullptr to the minWidthbits for errorPosition and crashed the minWidthbits code.   No matter what, the code should not crash.  Later, I realized I could use type->getNode().  I will add a CHECK_NULL.